src/algebra/smul_with_zero.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2021 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import group_theory.group_action.defs
+import algebra.algebra.basic
```

Comment:
This one would cause an import cycle.
```suggestion
```

Reply:
Although you might need to import `group_with_zero` in its place.

Reply:
I think that this comment is now outdated, since I have minimised the imports and I hope that there are no further loops.